string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,13 @@
+/* exported capitalizeWords */
+/* make the entire string lowercase;
+separate words by the spaces;
+after each space, capitalize the first letter only
+*/
+
+function capitalizeWords(string) {
+  var sentence = string.toLowerCase();
+  var words = sentence.split(' ');
+  for (var i = 0; i < words.length; i++) {
+    words[i] = words[i][0].toUpperCase() + words[i].substring(1);
+  } return words.join(' ');
```

Comment:
Make sure your `return` statement is always on its own line, it should not be on the same line as a closing brace.